test/__init__.py

```diff
@@ -317,7 +320,10 @@ def _connect(self, host, port, **kwargs):
         )
         try:
             try:
-                client.admin.command(HelloCompat.LEGACY_CMD)  # Can we connect?
+                if self.serverless:
+                    client.admin.command(HelloCompat.CMD)  # Can we connect?
+                else:
+                    client.admin.command("ping")  # Can we connect?
```

Comment:
We don't need to use the legacy command on serverless. We can use ping in all cases.